markdown/admin/logfiles.html.md.erb

```diff
@@ -0,0 +1,272 @@
+---
+title: HAWQ Administrative Log Files
+---
+
+Log files are files that include messages and other information about your HAWQ deployment, including the database and utilities.
+
+Every database instance in HAWQ \(master, standby, and segments\) runs a PostgreSQL database server with its own server log file. You generate log files when you invoke HAWQ management utilities directly, or indirectly via Ambari management operations. Additionally, other components in your HAWQ cluster (PXF, HDFS) generate log files of their own.
+
+These log files are distinctly located, formatted, configured, and managed.
+
+**Locating Log Files**
+
+HAWQ administrative log files reside in pre-defined or configured locations on the local file system of the HAWQ node.
+
+**Configuring Logging Parameters**
+
+Configurable logging parameters may affect what, when, and where messages are logged. You configure HAWQ administrative logging options via HAWQ server configuration parameters or command line options.
+
+**Managing Log Files**
+
+Log files are created and/or rotated at pre-defined or configured intervals.   Note that administrative log files are not automatically truncated or deleted. The administrator must implement and periodically run scripts to clean up these log files.
+
```

Comment:
Should this also contain "stub" sections for the "Log format" and "Examining" sections that follow?  maybe just roll the info into the intro paragraph and not into sections here?